internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -101,6 +101,20 @@ Resources:
           SigningBehavior: always
           SigningProtocol: sigv4
 
+  CloudFrontViewerRequestRewriteFunction:
+    Metadata:
+      'aws:copilot:description': 'CloudFront Function to rewrite viewer request for static site'
+    Type: AWS::CloudFront::Function
+    Properties: 
+      AutoPublish: true
+      FunctionCode: |
+        function handler(e){var i=e.request,n=i.uri;return n.endsWith("/")?i.uri+="index.html":n.includes(".")||(i.uri+="/index.html"),i}
```

Comment:
`n.endsWith("/")?i.uri+="index.html":n.includes(".")||(i.uri+="/index.html"),i` what is this syntax, can you name some names so that I can search up the terms and parse the code?